aspnet/mvc/overview/security/aspnet-mvc-5-app-with-sms-and-email-two-factor-authentication.md

```diff
@@ -100,36 +100,36 @@ This tutorial provides instructions for using either Twilio or ASPSMS but you ca
     [!code-csharp[Main](aspnet-mvc-5-app-with-sms-and-email-two-factor-authentication/samples/sample4.cs?highlight=3,16)]
 9. Run the app and log in with the account you previously registered.
 10. Click on your User ID, which activates the `Index` action method in `Manage` controller.  
-    ![](aspnet-mvc-5-app-with-sms-and-email-two-factor-authentication/_static/image3.png)
+    ![Screenshot that shows the A S P dot NET app Home page. USER ID is highlighted.](aspnet-mvc-5-app-with-sms-and-email-two-factor-authentication/_static/image3.png)
 11. Click Add.  
-    ![](aspnet-mvc-5-app-with-sms-and-email-two-factor-authentication/_static/image4.png)
+    ![Screenshot that shows the A S P dot NET app Account Settings page. None [Add] next to Phone Number section is highlighted.](aspnet-mvc-5-app-with-sms-and-email-two-factor-authentication/_static/image4.png)
```

Comment:
Since alt-text uses a screen reader, we don't need to place brackets around "Add."